packages/web-components/package.json

```diff
@@ -154,11 +158,8 @@
     "webpack": "^4.28.0"
   },
   "peerDependencies": {
-    "@carbon/ibmdotcom-services": "^1.8.0-rc.0",
-    "@carbon/ibmdotcom-utilities": "^1.8.0-rc.0",
     "carbon-components": "~10.11.0",
```

Comment:
@asudoh thank you for this!

I think we should also lock down carbon dependencies so we can manage for adopters.

Reply:
Makes sense @jeffchew - Merged in your change and updated.